sass/tokens/_compile.scss

```diff
@@ -0,0 +1,127 @@
+@use 'api';
+@use 'sass:map';
+@use 'sass:meta';
+
+
+/// # Compiling Maps
+/// Tools for compiling Accoutrement map tokens, 
+/// useful for exporting versions of 
+/// token maps with all of the Accoutrement syntax removed.
+/// @group token-compile
+
+/// Compile each token in a map
+/// ```
```

Comment:
@mirisuzanne I relocated these so that they would be `@example` annotations on their respective functions. Can you tell why the `map-compile-with` values are compiling to `null`?

```
/// @group token-compile
/// @example scss - Compile a map using a particular function
///   @use 'color';
///
///   $brand-colors: (
///     'brand-blue': hsl(234, 85%, 35%),
///     'dark-blue': '#brand-blue' ('shade': 30%),  
///   );
///
///   $fn: meta.get-function('color', $module: 'color');
///   /*! #{inspect(tools.map-compile-with($brand-colors, $fn))} */
///
```